index.html

```diff
@@ -928,31 +923,11 @@ <h2 class='informative'>Implementation Patterns</h2>
           <div class="example">
             <pre class="json">
 {
-  "@context": [
-    "https://www.w3.org/ns/activitystreams",
-    {
-      "date": {
-        "@id": "http://purl.org/dc/terms/date",
-        "@type": "xsd:dateTime"
-      }
-    }
-  ],
+  "@context": "https://www.w3.org/ns/activitystreams",
   "id": "urn:uuid:3c834a8f-5638-4412-aa4b-35ea80416a18",
   "type": "Create",
   "name": "Resource Creation",
-  "date": "2016-05-19T17:17:39Z",
-  "actor": [
```

Comment:
@birkland in this context, actors are simply taken from https://www.w3.org/TR/activitystreams-core/#actors; as such, the "minimal example" should show a "minimal" representation of an actor.

I will leave that change in place here unless someone specifically wants it moved into a separate PR.